addons/account_analytic_default/models/product.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    rules_count = fields.Integer(compute='_compute_rules_count', string='# Analytic Rules')
+
+    def _compute_rules_count(self):
+        Analytic = self.env['account.analytic.default']
+        for product in self:
+            product.rules_count = Analytic.search_count([('product_id', '=', product.id)])
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    rules_count = fields.Integer(compute='_compute_rules_count', string='# Analytic Rules')
+
+    def _compute_rules_count(self):
+        product_ids = self.mapped('product_variant_ids').ids
+        product_data = self.env['account.analytic.default'].read_group([('product_id', 'in', product_ids)], ['product_id'], ['product_id'])
+        for data in product_data:
+            self.rules_count += int(data['product_id_count'])
```

Comment:
`+=` ? Cool, the rules_count increase each time I evaluate this method.
`self`? so the same count is for every record?